mappings/net/minecraft/server/MinecraftServer.mapping

```diff
@@ -40,6 +40,8 @@ CLASS net/minecraft/server/MinecraftServer
 	FIELD field_33211 MAX_WORLD_BORDER_RADIUS I
 	FIELD field_33217 PLAYER_SAMPLE_UPDATE_INTERVAL J
 	FIELD field_33219 START_TICKET_CHUNKS I
+	FIELD field_33978 profileInstance Lnet/minecraft/server/MinecraftServer$a;
```

Comment:
Hmm, @sfPlayer1 did the matcher fail here?

Reply:
Yes, this has been fixed in an intermediary hotfix.

Reply:
@Shnupbups need to fix this then